src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java

```diff
@@ -80,15 +84,31 @@
 
   @ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "invalidInput")
```

Comment:
We don't need this anymore, from what I understand `handleConcreteRequestErrorMessageException` uses the value of the annotation at a class level for the custom exception throw.

Reply:
removed